spec/mailers/previews/instructor_notification_preview.rb

```diff
@@ -3,7 +3,7 @@
 
 class InstructorNotificationPreview < ActionMailer::Preview
   def message_to_instructors
-    InstructorNotificationMailer.email(example_alert)
+    InstructorNotificationMailer.email(example_alert, true)
```

Comment:
It looks like the intention was to add a second parameter to the mailer's email method, but it looks like this wasn't implemented. (And the fact that no test for it failed suggests that it's not being covered in the tests.) I would suggest using a named parameter for this, so that it's obvious what `true` means in this context. Or maybe this shouldn't have an extra parameter since the logic for it is implemented within the email method, and this preview method is just broken.

Reply:
Your right, thank you @ragesoss  for pointing this out, I think this is where my blocker has been, where the test I have been writing for this feature doesn't pass 😀,  since i was working and trying to fix the failed test in @prathamVaidya  PR,

This PR is still a WIP, as test to cover the feature has not yet been written.

Reply:
Are you still working on addressing this?